src/client/utils/hooks/useLockBodyScroll.ts

```diff
@@ -0,0 +1,20 @@
+import { useLayoutEffect } from 'react'
+
+type UseLockBodyScrollConfig = { lock: boolean }
+
+function useLockBodyScroll({ lock }: UseLockBodyScrollConfig = { lock: true }) {
```

Comment:
Another thing which is hard to know since we don't seem to have a guide in Notion with our coding conversations 😅

However, we prefer **arrow function** definitions.

Reply:
I see. I actually prefer function declarations but if that's the common practice among the team I'm ok with it :)